lib/foreman_remote_execution/engine.rb

```diff
@@ -138,6 +138,7 @@ class Engine < ::Rails::Engine
         end
 
         extend_rabl_template 'api/v2/smart_proxies/main', 'api/v2/smart_proxies/pubkey'
+        extend_rabl_template 'api/v2/interfaces/main', 'api/v2/remote_execution_features/interface_extensions'
```

Comment:
Usually the templates are named as `$what_is_being_extended/$with_what`. By reading `remote_execution_features/interface_extensions` I'd expect it to add information about interfaces to remote execution features. Probably `interfaces/execution_flag` woudl be a better name

Reply:
Makes sense, changed, thanks!